File: generateclasses/Prixcategorie_v.java
package entite;

public class Prixcategorie_v{
    int idcategorie; 
    String nomcategorie; 
    int idstyle; 
    String nomstyle; 
    int idtaille; 
    String nomtaille; 
    double prixtotal; 

    public Prixcategorie_v(){ }
    
    public int getIdcategorie(){
        return this.idcategorie;
    }
    public void setIdcategorie(int idcategorie){
        this.idcategorie=idcategorie;
    }
    public String getNomcategorie(){
        return this.nomcategorie;
    }
    public void setNomcategorie(String nomcategorie){
        this.nomcategorie=nomcategorie;
    }
    public int getIdstyle(){
        return this.idstyle;
    }
    public void setIdstyle(int idstyle){
        this.idstyle=idstyle;
    }
    public String getNomstyle(){
        return this.nomstyle;
    }
    public void setNomstyle(String nomstyle){
        this.nomstyle=nomstyle;
    }
    public int getIdtaille(){
        return this.idtaille;
    }
    public void setIdtaille(int idtaille){
        this.idtaille=idtaille;
    }
    public String getNomtaille(){
        return this.nomtaille;
    }
    public void setNomtaille(String nomtaille){
        this.nomtaille=nomtaille;
    }
    public double getPrixtotal(){
        return this.prixtotal;
    }
    public void setPrixtotal(double prixtotal){
        this.prixtotal=prixtotal;
    }

}

File: generateclasses/Prixcategorie_v.cs
namespace entite;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Table("prixcategorie_v")]
public class Prixcategorie_v : Motherobj<Prixcategorie_v>  {
    [Column("idcategorie")]
    public int idcategorie {get; set;} 
    [Column("nomcategorie")]
    public string nomcategorie {get; set;} 
    [Column("idstyle")]
    public int idstyle {get; set;} 
    [Column("nomstyle")]
    public string nomstyle {get; set;} 
    [Column("idtaille")]
    public int idtaille {get; set;} 
    [Column("nomtaille")]
    public string nomtaille {get; set;} 
    [Column("prixtotal")]
    public double prixtotal {get; set;} 

    public Prixcategorie_v(){ }
    

}
